code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -681,11 +686,27 @@ GLOBAL_VAR(bomb_set)
 	/// Is this a training disk?
 	var/training = FALSE
 
+/obj/item/disk/nuclear/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>You should keep this safe...</span>"
+
+/obj/item/disk/nuclear/examine_more(mob/user)
+	. = ..()
+	. += "Nuclear fission explosives are stored on all Nanotrasen stations in the system so that they may be rapidly destroyed, should the need arise."
+	. += ""
+	. += "Naturally, such a destructive capability requires robust safeguards to prevent accidental or malicious misuse. NT employs two mechanisms: an authorisation code from Central Command, \
+	and the nuclear authentication disk. Whilst the code is normally sufficient, enemies of Nanotrasen with sufficient resources may be able to spoof, steal, or otherwise crack the authorisation code. \
+	The NAD serves to protect against this. It is essentially a one-time pad that functions in tandem with the authorisation code to unlock the detonator of the fission explosive."
+
 /obj/item/disk/nuclear/unrestricted
 	name = "unrestricted nuclear authentication disk"
-	desc = "Seems to have been stripped of its safeties, you better not lose it."
+	desc = "A floppy disk containing unique cryptographic identification data. Used along with a valid code to detonate the on-site nuclear fission explosive."
 	restricted_to_station = FALSE
 
+/obj/item/disk/nuclear/unrestricted/examine(mob/user)
+	. = ..()
+	. += "<span class='danger'>This disk has had its safeties removed.</span>"
```

Comment:
Same here. And perhaps change this to describe what the safeties are.
```suggestion
	. += "<span class='warning'>This disk has had its safeties removed and it will no longer teleport back to the station.</span>"
```

Reply:
All done.